### DISCUSSION TITLE
GitHub Repo for Booklet Printing: Can't Resolve Merge Conflicts!

### DISCUSSION THREAD

**Discussion by Dorothy-Reyes** [pybamm-team/PyBaMM#3353] [Q&A]
Hi everyone,
I've been collaborating on a [booklet printing ](https://custompackaginglosangeles.com/booklet-printing/)project using GitHub but we keep encountering merge conflicts slowing us down. How can we effectively resolve these conflicts to streamline our workflow? Seeking guidance from experienced [GitHub](https://github.com/) users!

**Comment by brosaplanella** [comment]
Hi! Your questions are about Github rather than PyBaMM, so you should post them in the relevant Github channels (e.g. Stack Overflow). The [GitHub docs](https://docs.github.com/en) might also be useful.
